mosec/coordinator.py

```diff
@@ -159,19 +162,31 @@ def run(self):
             self.coordinate()
 
     def get_decoder(self) -> Callable[[bytes], Any]:
-        if STAGE_INGRESS in self.worker._stage:
+        """get the decoder function for this stage
```

Comment:
Does this docstring pass the lint? To me this looks not well-formatted.

Reply:
BTW, I think previously we ignore many docstring issues (many places we dont have docstring or format not that correct), maybe we can fix all here or have a separate MR to fix it later?

Reply:
Tried but it seems this config doesn't work. ❓ 

```toml
default-docstring-type = "google"
```

Reply:
Or we could use `pydocstyle ${PROJECT_PATH} --convention=google`?

Reply:
Can. I just wonder why PyCQA has so many tools but doesn't integrate them together.